File: check_for_palindromes/palindrome.js
/**
 * @func palindrome
 * Checks if str is a palindrome
 * @param {String} str
 * @returns {Boolean} true if str is a palindrome otherwise, false
 */
function palindrome(str) {
    // Get rid of non alphanumeric characters
    var cleanedString = str.toLowerCase().replace(/[^a-z0-9]/g, "");
    // Compare the cleaned string to iteself in reverse, return true or false
    return ( cleanedString === cleanedString.split("").reverse().join("") );
}
palindrome("Rac_*& eCar")

File: check_for_palindromes/palindrome.py
def palindrome(string):
    """
    Check to see if string is a palindrome
    Returns a Boolean
    """
    import re 
    # Get rid of non alphabets or non numbers characters
    cleanedString = re.sub("[\W_]", "", string).lower()
    
    # Reverse and compare
    return cleanedString[::-1] == cleanedString
   
print palindrome("My age is 0, 0 si ega ym.")